main_page.py

```diff
@@ -0,0 +1,31 @@
+from selenium.webdriver.common.by import By
+from BaseClass.base_class import BaseClass
+from time import sleep
+
+
+class MainPage:
+    website = "http://www.amazon.com/"
+    MAIN_PAGE_CONTROL = (By.ID, 'gw-card-layout')
+    SIGN_IN_HOVER = (By.ID, 'nav-link-accountList')
+    LOGIN_PAGE_NAVIGATE = (By.CLASS_NAME, 'nav-action-inner')
+    SEARCH_BAR = (By.ID, 'twotabsearchtextbox')
+    SEARCH_BUTTON = (By.ID, 'nav-search-submit-button')
+    word_to_search = "samsung"
+
+    def __init__(self, driver):
+        self.driver = driver
+        self.functions = BaseClass(self.driver)
+
+    def load_main_page(self):
+        self.driver.get(self.website)
+        main_page_loaded = self.functions.exist_element(self.MAIN_PAGE_CONTROL)
+        assert main_page_loaded, True

```

Comment:
Assert kullanımı yanlış. Virgülden sonra anlamlı bir mesaj yazalım. ("Mainpage is not reachable" gibi)